src/Plugin/rest/resource/ReactParagraphsResource.php

```diff
@@ -129,13 +129,11 @@ public function get($entity_type_id, $bundle) {
       $field_config = $field_config_storage->load("$entity_type_id.$bundle.$field_name");
       if ($field_config && ($plugin = $this->getReactFieldsPlugin($field_config))) {
         $data[$field_name] = $plugin->getFieldInfo($form[$field_name], $field_config);
-        $this->moduleHandler->invokeAll(
-          "react_paragraphs_getfieldinfo_post_alter",
-          [
-            $form[$field_name],
-            $field_config,
-            $data[$field_name],
-          ]
+        $this->moduleHandler->alter(
+          'react_paragraphs_form_field_data',
+          $data[$field_name],
+          $form[$field_name],
+          $field_config
```

Comment:
This order changed. Will have to swap it in stanford_news.

Reply:
yeah. it seemed more inline with other alter hooks. the first argument is the one you wanna alter

Reply:
I'm onboard with that. Thanks for quality control.